packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -250,7 +250,7 @@ export default function ClassifierContainer({
             subjectID={subjectID}
             workflowSnapshot={workflowSnapshot}
           /> :
-          <Paragraph>Loading…</Paragraph>
+          <Paragraph>Loading the Classifier</Paragraph>
```

Comment:
This is just to differentiate "loading" messages between app-root loading the Classify page, lib-classifier loading the entire Classifier component, and parts of the classifier with messages like "Loading a subject".

Eventually the loading UX of lib-classifier will be enhanced with skeleton components, loading animations, polite live regions, etc. See https://github.com/zooniverse/front-end-monorepo/issues/6484